### ISSUE TITLE
Does not compile

### ISSUE THREAD

**Issue by euskadi31** [jurre/TravisToday#3]
Hi, 

I try compile with Xcode Version 6.4 (6E35b)

> TravisToday/Today/TravisViewController.swift:39:49: Operand of postfix '!' should have optional type; type is 'String'
> TravisToday/Today/TravisViewController.swift:43:43: 'NSMutableArray' is not implicitly convertible to '[AnyObject]'; did you mean to use 'as' to explicitly convert?
> TravisToday/Today/TravisViewController.swift:93:30: Cannot invoke 'find' with an argument list of type '(RepoConfig, completion: (Bool, Repo?) -> Void)'

**Comment by jurre** [comment]
yeah this is still on an older swift version. Hope to get around to updating it soon :)

**Comment by euskadi31** [comment]
ok :)
